.canon.yaml

```diff
@@ -0,0 +1,8 @@
+viam-cartographer:
+    image_amd64: ghcr.io/viamrobotics/canon:amd64
+    image_arm64: ghcr.io/viamrobotics/canon:arm64
```

Comment:
Same question as https://github.com/viamrobotics/viam-orb-slam3/pull/25#discussion_r1148026779 (I assume the answer will be the same)